tsconfig.json

```diff
@@ -1,15 +1,16 @@
 {
-	"compilerOptions": {
-		"module": "commonjs",
-		"declaration": true,
-		"noImplicitAny": true,
-		"removeComments": true,
-		"preserveConstEnums": true,
-		"strictNullChecks": true,
-		"sourceMap": true,
-		"target": "es6",
-		"types": [ "bluebird-global" ],
-		"noUnusedParameters": true,
-		"noUnusedLocals": true
-	}
+    "compilerOptions": {
+        "module": "commonjs",
+        "declaration": true,
+        "noImplicitAny": true,
+        "removeComments": true,
+        "preserveConstEnums": true,
+        "strictNullChecks": true,
+        "sourceMap": true,
+        "target": "es6",
+        "baseUrl": "./typings",
```

Comment:
I followed from [this](https://github.com/Microsoft/TypeScript/issues/12222) thread, as I was originally attempting to use `typeRoots`.

Adding it to the gulp file appears to cause Visual Code to complain as it doesn't find them. I'll use `includes` if that's better than `baseUrl`.